utilities/pipelines/dependencies/constructs/StoreVhdToStorage/deploy.bicep

```diff
@@ -0,0 +1,139 @@
+targetScope = 'subscription'
+
+// ================ //
+// Input Parameters //
+// ================ //
+
+// RG parameters
+@description('Optional. The name of the resource group to deploy')
+param resourceGroupName string = 'validation-rg'
```

Comment:
Question: 

Before we leveraged the pipeline variable to set the RG-name. Now it's implemented here and not passed from the outside - which means the user would need to maintain 2 places (which should not be an issue using search & replace).

-> Should we pass the variable as a paramter from the pipeline?

Reply:
Updated value with the `<<resourceGroupName>>` token